modules/apps/client-extension/client-extension-type-item-selector-web/src/main/java/com/liferay/client/extension/type/item/selector/web/internal/item/selector/CETItemSelectorViewDescriptor.java

```diff
@@ -105,7 +106,17 @@ public boolean isShowBreadcrumb() {
 
 	private Predicate<CET> _getPredicate(String type) {
 		if (Objects.equals(
-				type, ClientExtensionEntryConstants.TYPE_GLOBAL_JS)) {
+				type, ClientExtensionEntryConstants.TYPE_GLOBAL_CSS)) {
+
+			return cet -> {
+				GlobalCSSCET globalJSCET = (GlobalCSSCET)cet;
```

Comment:
typo in variable name. It should be `globalCSSCET`, right?

Reply:
Ops, will fix it, thanks